### PR TITLE
Create thematic-analysis.md

### LINKED ISSUE #4903: [Term Entry] UI/UX Qualitative Research: Thematic Analysis
### Reason/inspiration (optional)

We would like a new entry on the `Thematic Analysis` term under UI/UX. The entry should go in a new file under `docs/content/uiux/concepts/qualitative-research/terms/thematic-analysis/thematic-analysis.md`.

The entry should include:

- An introduction to the term and how it is used within UI/UX
- Image(s) to aid understanding (if applicable)

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by sturrdlefish**
### Description

A term entry on thematic analysis has been created at docs/content/uiux/concepts/qualitative-research/thematic-analysis/thematic-analysis.md.

### Issue Solved

Closes #4903 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/uiux/concepts/qualitative-research/thematic-analysis/thematic-analysis.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Thematic analysis** is done by examining qualitative data (e.g. results from user interviews or feedback forms) and identifying patterns within it, categorizing it using **codes** and using those codes to determine **themes**. This results in a clear set of actionable insights to be worked on.
+
+## Codes
+Codes are used to identify common topics of interest, functioning as a tag for categorizing data. For example, a thematic analysis of feedback for a messaging app may include codes such as "interface", "functionality", "accessibility", and so on, documenting the comments made about each aspect of the application.
+
+## Themes
+Once the data has been categorized, themes can be determined within the created codes. A theme is a core idea that is obtained through observing similar concepts within a code, indicating something about user needs or preferences and providing meaningful insights.
```
Comment (SaviDahegaonkar) [inline content/uiux/concepts/qualitative-research/thematic-analysis/thematic-analysis.md:20]:
```suggestion
## Themes
Once the data has been categorized, themes can be determined within the created codes. A theme is a core idea that is obtained through observing similar concepts within a code, indicating something about user needs or preferences, and providing meaningful insights.
```

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @sturrdlefish ,
I looked at your file, and I find it absolutely correct, so I hereby approve it for the first review.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @sturrdlefish thank you for your contribution, the entry is nicely written! 😃 

I've suggested a few changes, could you please review and modify them at your earliest convenience? Thank you! 😄

File: content/uiux/concepts/qualitative-research/thematic-analysis/thematic-analysis.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Thematic analysis** is done by examining qualitative data (e.g. results from user interviews or feedback forms) and identifying patterns within it, categorizing it using **codes** and using those codes to determine **themes**. This results in a clear set of actionable insights to be worked on.
```
[the remark this answers is not in the document]
Comment (sturrdlefish) [reply content/uiux/concepts/qualitative-research/thematic-analysis/thematic-analysis.md]: Added! All changes have now been completed

**Review by mamtawardhani — APPROVED** [review]
Thank you for your contribution @sturrdlefish. The entry looks good to be merged 😄 🎊

**Comment by mamtawardhani** [comment]
Hey @sturrdlefish, we'll need a small change from your side. The current path is wrong, please change it to:
`docs/content/uiux/concepts/qualitative-research/terms/thematic-analysis/thematic-analysis.md`
You are missing the terms folder. Please do this at your earliest. Thank you! 😃

**Comment by sturrdlefish** [comment]
@mamtawardhani Fixed! :)

**Pull request merged**